tools/operator/testclient/operations/invokequery.go

```diff
@@ -243,28 +272,28 @@ func (i InvokeQueryUIObject) invokeQueryTransactions(invokeQueryObjects []Invoke
 			err := i.invokeConfig(invokeQueryObjects[invokeQueryObjectIndex].ChannelOpt.Name, args)
 			if err != nil {
 				logger.ERROR("Failed to complete invokes/queries")
-				statusError = fmt.Errorf("Something went wrong in completing invokes/queries")
-			}
-			blockchain, err := i.fetchMetrics(invokeQueryObjects[invokeQueryObjectIndex])
-			if err != nil {
-				logger.ERROR("failed fetching metrics")
-				statusError = fmt.Errorf("Something went wrong in fetching metrics")
-			}
-			var blockchainHeight int
-			var transactionCount int
-			var blockHash string
-			channelBlock := blockchain[invokeQueryObjects[invokeQueryObjectIndex].ChannelOpt.Name]
-			for key, value := range channelBlock {
-				if blockchainHeight == 0 && transactionCount == 0 {
-					blockchainHeight = value.peerBlockchainHeight
-					transactionCount = value.peerTransactionCount
-					blockHash = value.peerBlockHash
-				} else {
-					if value.peerBlockchainHeight != blockchainHeight || value.peerTransactionCount != transactionCount || value.peerBlockHash != blockHash {
-						logger.ERROR("Peers are not in sync")
-						statusError = fmt.Errorf("Something went wrong with peer %s as blockchain height or transactions does not match up", key)
+				blockchain, err := i.fetchMetrics(invokeQueryObjects[invokeQueryObjectIndex])
```

Comment:
By changing this to check all the peers ONLY when invokeConfig fails, it seems to weaken the check. Don't we want every peer to be in sync for every test? What if there is a problem with a peer that is not being used for the invokes? You would miss it, if you make this change.

Reply:
fixed